GPL/nativeBuildProperties.gradle

```diff
@@ -52,6 +52,13 @@ model {
 				}
 			}
 		}
+		if (isCurrentWindows()) {
```

Comment:
I think would be better placed up in the `toolChains` block.  That's where all the other toolchain workarounds live.